zerver/webhooks/sentry/doc.md

```diff
@@ -20,6 +20,11 @@ you like. If you want notifications for issues and not just events,
 you can scroll down to **Webhooks** on the same page and check the
 box that says **issues**. Make sure that you set up the permissions
 so that the integration will visible to the right people.
+**NOTE**: Zulip also supports configuring this as a webhook in Sentry &mdash;
+which, while easier to configure (Navigate to **Settings > Integrations >
+WebHooks**) may not include the full breadth of event types.
```

Comment:
Can we expand what "not include the full breadth of event types" means for the user?  Like, will some events be dropped entirely, or will the events be less complete, or what?

Reply:
Yeah, I'll change that currently only notifications for errors are handled. Issue assignment events, for example, don't get triggered for this type of configuration. In future there are plans for the "full integration" to support other types of events like project changes, etc.